src/DynamoCore/Library/DocumentationServices.cs

```diff
@@ -58,10 +58,8 @@ public static XmlReader GetForAssembly(string assemblyPath, IPathManager pathMan
                 assemblyLocation = cashedAssemblyLocation;
             }
 
-            var assemblyPath = Path.GetFullPath(assemblyLocation);
-
-            var baseDir = Path.GetDirectoryName(assemblyPath);
-            var xmlFileName = Path.GetFileNameWithoutExtension(assemblyPath) + ".xml";
+            var baseDir = pathManager.DynamoCoreDirectory;
+            var xmlFileName = Path.GetFileNameWithoutExtension(assemblyLocation) + ".xml";
```

Comment:
Suppose `assemblyLocation` was the correct location for the assembly. Why would we want to simply blow away the path like this, replacing it with `DynamoCore`'s directory? How will that affect other assemblies loaded from other directories (like user node libraries, Extensions or Packages)?

If your above statement is correct, it sounds like `assemblyLocation` is being passed a path obtained from `Assembly.GetExecutingAssembly` (in Dynamo standalone this would be `DynamoSandbox`, but in Revit it's the Revit install directory). 

I also noticed that directly above your fix is another fix, specifically for operators - is that unrelated?